requirements-dev.txt

```diff
@@ -1,3 +1,3 @@
+setuptools==50.3.2
```

Comment:
I still think we should only pin on minor releases... ;-)

Reply:
With dependabot the cost of upgrade is close to zero and pinning to patch version guarantees ~100% reproduction of the process.